.github/workflows/preview.yml

```diff
@@ -6,7 +6,7 @@ on:
     types: [opened, synchronize, reopened, closed]
     paths:
       - 'docs/**'
-      - '*.md'
+      - '**.md'
```

Comment:
Just curious, what's the difference between *.MD and **.MD here?

Reply:
*.md  => Matches all .md files at the **root** of the repository.
**.md  => Matches all .md files in the repository.

<img width="757" alt="image" src="https://user-images.githubusercontent.com/35299017/166622549-c36b3d98-bbc5-4ccc-84d0-f441313a87d6.png">

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths